magenta/models/rl_tuner/unpack_ckpt_from_bundle.py

```diff
@@ -0,0 +1,37 @@
+r"""Code to extract a tensorflow checkpoint from a bundle file.
```

Comment:
I would just call this unpack_bundle (because it unpacks the metagraph too), and I think it would make sense for this to live in our /scripts folder because it's generally useful.